### DISCUSSION TITLE
There is already an open DataReader associated with this Connection which must be closed first

### DISCUSSION THREAD

**Discussion by vikasmohal** [maxtoroq/DbExtensions#63] [Q&A] [answered]
**Is there any way to create multiple Recordset for each Include....**

An unhandled exception of type 'MySql.Data.MySqlClient.MySqlException' occurred 

Additional information: There is already an open DataReader associated with this Connection which must be closed first.

**Comment by maxtoroq** [comment, marked as answer]
If the provider doesn't support MARS, you can only eager-load a one-to-many association for a single entity, e.g. using Single or First.
